### ISSUE TITLE
After Updating to Latest Version 24hrs and Current data Chat view is Not shown

### ISSUE THREAD

**Issue by mskian** [mikecao/umami#74]
After Updating to `0.17.0`
24hrs and Current data Chat view is Not shown

![Screenshot from 2020-08-29 08-17-42](https://user-images.githubusercontent.com/10300271/91626856-37642200-e9d0-11ea-942b-ad6576b0d405.png)  

![Screenshot from 2020-08-29 08-18-04](https://user-images.githubusercontent.com/10300271/91626858-38954f00-e9d0-11ea-94eb-133189d99c08.png)

**Comment by mikecao** [comment]
Nothing changed in the tracker or collection code. What do you see in the network panel on your website?

**Comment by mskian** [comment]
Tracker, realtime, data collections are working well
no errors on console log
the issue only on chat view (the current day and 24 hours)

**Comment by mikecao** [comment]
Can you send the queries from the umami dashboard? Like pageviews? Also, are you using MySQL?

**Comment by mskian** [comment]
are you using MySQL? - yes

Select today
```
2020-08-29T10:01:16.728380Z	 6244 Connect	root@localhost on analytics using TCP/IP
2020-08-29T10:01:16.728545Z	 6244 Query	SELECT @@socket
2020-08-29T10:01:16.728852Z	 6245 Connect	root@localhost on analytics using Socket
2020-08-29T10:01:16.729005Z	 6245 Query	SELECT @@max_allowed_packet
2020-08-29T10:01:16.729146Z	 6245 Query	SELECT @@wait_timeout
2020-08-29T10:01:16.729257Z	 6244 Quit	
2020-08-29T10:01:16.729321Z	 6245 Query	SELECT @@max_allowed_packet
2020-08-29T10:01:16.729417Z	 6245 Query	SELECT @@wait_timeout
2020-08-29T10:01:16.729529Z	 6245 Query	SELECT 1
2020-08-29T10:01:16.730088Z	 6245 Prepare	select DATE_FORMAT(convert_tz(created_at,'+00:00','+05:30'), '%Y-%m-%d %H:00:00') t,
        count(distinct session_id) y
      from pageview
      where website_id=?
      and created_at between ? and ?
      group by 1
      order by 1
2020-08-29T10:01:16.730207Z	 6245 Execute	select DATE_FORMAT(convert_tz(created_at,'+00:00','+05:30'), '%Y-%m-%d %H:00:00') t,
        count(distinct session_id) y
      from pageview
      where website_id=1
      and created_at between '2020-08-28 18:30:00' and '2020-08-29 18:29:59.999000'
      group by 1
      order by 1
2020-08-29T10:01:16.771036Z	 6246 Connect	root@localhost on analytics using TCP/IP
2020-08-29T10:01:16.771178Z	 6225 Quit	
2020-08-29T10:01:16.771371Z	 6246 Query	SELECT @@socket
2020-08-29T10:01:16.771795Z	 6247 Connect	root@localhost on analytics using Socket
2020-08-29T10:01:16.771926Z	 6247 Query	SELECT @@max_allowed_packet
2020-08-29T10:01:16.772053Z	 6247 Query	SELECT @@wait_timeout
2020-08-29T10:01:16.772146Z	 6246 Quit	
2020-08-29T10:01:16.772281Z	 6247 Query	SELECT @@max_allowed_packet
2020-08-29T10:01:16.772398Z	 6247 Query	SELECT @@wait_timeout
2020-08-29T10:01:16.772520Z	 6247 Query	SELECT 1
2020-08-29T10:01:16.772904Z	 6247 Prepare	select DATE_FORMAT(convert_tz(created_at,'+00:00','+05:30'), '%Y-%m-%d %H:00:00') t,
        count(*) y
      from pageview
      where website_id=?
      and created_at between ? and ?
      group by 1
      order by 1
2020-08-29T10:01:16.772960Z	 6247 Execute	select DATE_FORMAT(convert_tz(created_at,'+00:00','+05:30'), '%Y-%m-%d %H:00:00') t,
        count(*) y
      from pageview
      where website_id=1
      and created_at between '2020-08-28 18:30:00' and '2020-08-29 18:29:59.999000'
      group by 1
      order by 1
```

select 24 hrs

```
2020-08-29T09:59:43.036701Z	 6222 Connect	root@localhost on analytics using TCP/IP
2020-08-29T09:59:43.036878Z	 6222 Query	SELECT @@socket
2020-08-29T09:59:43.037216Z	 6223 Connect	root@localhost on analytics using Socket
2020-08-29T09:59:43.037334Z	 6223 Query	SELECT @@max_allowed_packet
2020-08-29T09:59:43.037433Z	 6223 Query	SELECT @@wait_timeout
2020-08-29T09:59:43.037516Z	 6222 Quit	
2020-08-29T09:59:43.037599Z	 6223 Query	SELECT @@max_allowed_packet
2020-08-29T09:59:43.037677Z	 6223 Query	SELECT @@wait_timeout
2020-08-29T09:59:43.037926Z	 6223 Prepare	select sum(t.c) as "pageviews",
        count(distinct t.session_id) as "uniques",
        sum(case when t.c = 1 then 1 else 0 end) as "bounces",
        sum(t.time) as "totaltime"
      from (
         select session_id,
           DATE_FORMAT(created_at, '%Y-%m-%d %H:00:00'),
           count(*) c,
           floor(unix_timestamp(max(created_at)) - unix_timestamp(min(created_at))) as "time"
         from pageview
         where website_id=?
         and created_at between ? and ?
         group by 1, 2
     ) t
2020-08-29T09:59:43.038024Z	 6223 Execute	select sum(t.c) as "pageviews",
        count(distinct t.session_id) as "uniques",
        sum(case when t.c = 1 then 1 else 0 end) as "bounces",
        sum(t.time) as "totaltime"
      from (
         select session_id,
           DATE_FORMAT(created_at, '%Y-%m-%d %H:00:00'),
           count(*) c,
           floor(unix_timestamp(max(created_at)) - unix_timestamp(min(created_at))) as "time"
         from pageview
         where website_id=1
         and created_at between '2020-08-28 10:30:00' and '2020-08-29 10:29:59.999000'
         group by 1, 2
     ) t
2020-08-29T09:59:43.042793Z	 6219 Quit
```

**Comment by mskian** [comment]
install umami on august 25 
server - Hetzner Cloud VPS
Mysql 8
full site pageview and visitors result

![Screenshot from 2020-08-29 15-36-27](https://user-images.githubusercontent.com/10300271/91634369-83818780-ea0d-11ea-9976-425f12421b65.png)

**Comment by mikecao** [comment]
I think it may be an issue with how the timezone is calculated. But you should still be recording data so that's good. I'll work on a fix.

**Comment by mikecao** [comment]
@mskian This should be fixed in the latest version v0.19.0

**Comment by mskian** [comment]
@mikecao Thanks now it shows the graph on chat
